sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/parquet/ParquetTable.java

```diff
@@ -122,4 +124,11 @@ public IsBounded isBounded() {
   public ProjectSupport supportsProjects() {
     return ProjectSupport.WITH_FIELD_REORDERING;
   }
+
+  private String resolveFilePattern(String location) {
+    if (location.endsWith("/")) {
```

Comment:
Previously user can provide a directory without trailing "/", now the runtime FileSystems.match() will resolve to not found or a directory (depending on filesystem type), we may mention this breaking change in CHANGES.md